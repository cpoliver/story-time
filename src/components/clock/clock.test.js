import React from 'react';
import renderer from 'react-test-renderer';

import Clock from './index';

it('renders correctly', () => {
    const tree = renderer.create(
      <Clock hours={3} minutes={12} seconds={45} />
    );

    expect(tree).toMatchSnapshot();
});
