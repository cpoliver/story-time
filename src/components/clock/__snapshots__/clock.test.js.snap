exports[`test renders correctly 1`] = `
<div
  className="clock">
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-round" />
  <div
    className="clock-hour-marker-triangle" />
  <div
    className="clock-hour-marker-baton" />
  <div
    className="clock-hour-marker-baton" />
  <div
    className="clock-inner" />
  <div
    className="clock-second-marker clock-second-marker-bold"
    style={
      Object {
        "transform": "rotate(0deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(6deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(12deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(18deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(24deg)",
      }
    } />
  <div
    className="clock-second-marker clock-second-marker-bold"
    style={
      Object {
        "transform": "rotate(30deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(36deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(42deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(48deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(54deg)",
      }
    } />
  <div
    className="clock-second-marker clock-second-marker-bold"
    style={
      Object {
        "transform": "rotate(60deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(66deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(72deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(78deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(84deg)",
      }
    } />
  <div
    className="clock-second-marker clock-second-marker-bold"
    style={
      Object {
        "transform": "rotate(90deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(96deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(102deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(108deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(114deg)",
      }
    } />
  <div
    className="clock-second-marker clock-second-marker-bold"
    style={
      Object {
        "transform": "rotate(120deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(126deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(132deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(138deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(144deg)",
      }
    } />
  <div
    className="clock-second-marker clock-second-marker-bold"
    style={
      Object {
        "transform": "rotate(150deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(156deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(162deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(168deg)",
      }
    } />
  <div
    className="clock-second-marker"
    style={
      Object {
        "transform": "rotate(174deg)",
      }
    } />
  <div
    className="clock-center" />
  <div
    className="clock-hand clock-hand-hour"
    style={
      Object {
        "transform": "rotate(90deg)",
      }
    } />
  <div
    className="clock-hand clock-hand-minute"
    style={
      Object {
        "transform": "rotate(72deg)",
      }
    } />
  <div
    className="clock-hand clock-hand-second"
    style={
      Object {
        "transform": "rotate(270deg)",
      }
    } />
</div>
`;
